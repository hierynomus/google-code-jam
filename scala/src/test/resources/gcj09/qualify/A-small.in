10 25 10
aglptuyfqr
qjvzwlutoj
vjucrcjtjs
vjygbtczzu
trdoqzbkup
uqyoussdnb
wikwrlqtnp
oigtwwtxjn
mffmvaenfs
obcaytoopb
ffjlyhjtgs
juxqgsqlhy
fvzhyzdmps
puzyhzhrtn
lejjupdncl
lhimpgouuf
mjcnjjmqch
foloxgecvg
pgqauhgibb
opkscubqme
ysslirtoto
xlwnlrutbv
urmhjodner
rellqucjkx
yhkwwvpqpr
nlkzuwpxjw
oigtwwt(nvxh)jn
(otd)(fmn)(wfh)(imh)(bgv)(ehu)(vyc)(yps)(lsk)(uwx)
(xflt)(uabl)(wh)(ntwf)(xhl)(rzdp)(iluy)(stfm)(bjmn)(hkvx)
(ghijklmnoprstuvwxyzabcdef)(efgijklmnoqrstuvxyabcd)(vwxyabcdeghijklmnopqrtu)(cdefhijklmnopqrtuvwyzab)(eghjklmnprstuvwxyzabc)(suwxyzabefghijklmnpqr)(orstuwxyzacdfghijlmn)(ijklmnopqrstuvwxyzabcdefg)(nopqrstuvwxyzabcdefgklm)(yzacdefgijklpqrsuvw)
(tuvwxyzacdefghijklmopqrs)(zadefghiklmnoqrsuvwxy)(qrstuvwxyzacefghjklmnop)(rstuvxzacdefhijklmnpq)(pqrstvwyzabcdfghijklmno)(vwxyabcefghjklmnopqrstu)(mnpqstuvwxyzabcdfghijk)(vwxyabdefghijklmnopqst)(uvxyzabcdefgijklnopqrst)(jkloprstuvwxyzabcdefgh)
(iowx)(mbdf)(chnw)(alrt)(twyg)(jmta)(dopy)(opu)(fimp)(sbci)
(vwxyzabcdefghijlmnopqrtu)(hjklmnoprstuxyzabcdf)(bdefhjklmnopqrsuvwxyz)(ijklopqrstuvxyzabcdefgh)(ghijlmnopqrstvxyzbcde)(cdegijklmnoqrstuvwxyzab)(rsuvwxzabdefgijklmnpq)(abcfhijklmopqrtuvwxyz)(cdeghjklmnopqrstuvwxyza)(klmopqstuvwxyzabcdefgh)
(fmor)(hjot)(ehln)(owj)(qxae)(ghl)(bef)(ciln)(fkuv)(gnow)
(stuvwyzacdfghikmopqr)(ijklmnopqrtuvwxzabefgh)(qrtuwyzacdefghiklmno)(jlmnopqrstuwxyzabcdeghi)(wxyzbcdefghijlmnoprstu)(pqrstuvwxyzabcdefgjkmn)(mnopqstuvwxyacdefghikl)(fhijklmnopqrsuwxyzabcde)(ijkmnopqrstuvwxyzabcdef)(efghijklmnopqrstuvwxyzbd)
