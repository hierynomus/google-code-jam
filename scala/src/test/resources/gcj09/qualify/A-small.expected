Case #1: 0
Case #2: 1
Case #3: 0
Case #4: 1
Case #5: 4
Case #6: 6
Case #7: 1
Case #8: 5
Case #9: 1
Case #10: 10